sycl/test-e2e/Matrix/element_wise_all_ops_scalar.cpp

```diff
@@ -7,6 +7,9 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: aspect-ext_intel_matrix, gpu
 
+// https://github.com/intel/llvm/issues/14826
```

Comment:
I suggest adding this instead: `// REQUIRES-INTEL-DRIVER: lin: 30049`

Reply:
Fixed in 7b630036a17c3f346f7be74077e76f01c3caf9ea